src/components/movies/movieTopRated.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <moviesMix :moviesEndpoint="'top_rated'"></moviesMix>
```

Comment:
Use `PascalCase` for component names. Also refactor to `moviesEndpoint="top_rated"`